lib/modules/manager/pipenv/artifacts.spec.ts

```diff
@@ -13,19 +13,23 @@ import { GlobalConfig } from '../../../config/global';
 import type { RepoGlobalConfig } from '../../../config/types';
 import * as docker from '../../../util/exec/docker';
 import type { StatusResult } from '../../../util/git/types';
+import { find as _find } from '../../../util/host-rules';
 import { getPkgReleases as _getPkgReleases } from '../../datasource';
 import * as _datasource from '../../datasource';
 import type { UpdateArtifactsConfig } from '../types';
-import * as pipenv from '.';
+import * as _pipenv from '.';
 
 const datasource = mocked(_datasource);
+const pipenv = mocked(_pipenv);
```

Comment:
wrong, shouldn't be mocked. import the real mocked file

Reply:
Could you suggest how it should be done, as I'm new to TypeScript?

Reply:
@viceice @rarkins could you suggest how it should be done, as I'm new to TypeScript?

Reply:
I hope I implemented it correctly this time